ci/test/docs-widgets/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "docs-widgets-tests",
+  "description": "Tests for the docs-widgets",
+  "scripts": {
+    "test": "vitest"
+  },
+  "dependencies": {
+    "jsdom": "^24.0.0"
+  },
+  "devDependencies": {
+    "vitest": "^1.4.0"
```

Comment:
nit: but jsdom and vitest should both be the same type of dependency. Since this package is only used for testing, it would be reasonable to not use dev dependencies at all.

Reply:
Ah yes! Good catch!